File: 2380-time-needed-to-rearrange-a-binary-string/2380-time-needed-to-rearrange-a-binary-string.java
class Solution {
    public int secondsToRemoveOccurrences(String s) {
        int cnt=0;
        while(true){
            int idx=s.indexOf("01");
            if(idx==-1)break;
            else{
                s=s.replaceAll("01","10");
                cnt++;
            }
        }
        
        return cnt;
    }
}

File: 2380-time-needed-to-rearrange-a-binary-string/2380-time-needed-to-rearrange-a-binary-string.cpp
class Solution {
public:
    int secondsToRemoveOccurrences(string s) {
        int flag = 0;
        int time = 0;
        
        for(int i=0;i<s.length()-1;i++)
        {
           for(int j = 0 ; j<s.length() ; j++)
           {
               if(s[j] == '0' && s[j+1] == '1')
               {
                   swap(s[j],s[j+1]);
                   ++j;
                   // time++;/
                   flag = 1;
               }
           } 
            if(flag == 1){
                time++;
            }
            flag = 0;
        }
        
        return time;
    }
};